server/storage/mvcc/kvstore_test.go

```diff
@@ -542,23 +542,19 @@ type hashKVResult struct {
 func TestHashKVWhenCompacting(t *testing.T) {
 	b, tmpPath := betesting.NewDefaultTmpBackend(t)
 	s := NewStore(zaptest.NewLogger(t), b, &lease.FakeLessor{}, StoreConfig{})
-	defer s.Close()
-	defer b.Close()
-	defer os.Remove(tmpPath)
+	defer cleanup(s, b, tmpPath)
 
 	rev := 10000
 	for i := 2; i <= rev; i++ {
 		s.Put([]byte("foo"), []byte(fmt.Sprintf("bar%d", i)), lease.NoLease)
 	}
 
 	hashCompactc := make(chan hashKVResult, 1)
-
 	donec := make(chan struct{})
-	var wg sync.WaitGroup
```

Comment:
It seems it would better to keep this WaitGroup, because we need to make sure all goroutines have already finished their tasks before exiting the test case.

Reply:
Thank you for taking a look @ahrtr 
I see what you mean, I will adjust this.

Reply:
@ahrtr : I have added the WaitGroup back, and would need your approval for the test workflows to run again, please.
Thank you.